<launch>
        <rosparam command="load" file="$(find task_priority)/launch/merbots.yaml" />

        <node name="multi_task_priority" pkg="task_priority" type="multi_task_priority" output="screen" respawn="false"/>
<node name="grasp_srv_server" pkg="merbots_grasp_srv" type="grasp_srv_server" output="screen" respawn="false"/>
</launch>
